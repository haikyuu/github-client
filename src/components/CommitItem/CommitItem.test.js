import React from 'react'
import renderer from 'react-test-renderer'
import { CommitItem } from '@components'
const commit = {
    "sha": "848593c0f0b75da9ff342596908c5091a2cd1430",
    "commit": {
      "author": {
        "name": "Brian Vaughn",
        "email": "bvaughn@fb.com",
        "date": "2017-03-25T07:37:51Z"
      },
      "committer": {
        "name": "Facebook Github Bot",
        "email": "facebook-github-bot@users.noreply.github.com",
        "date": "2017-03-25T07:46:54Z"
      },
      "message": "Moved takeSnapshot method from UIManager to ReactNative\n\nReviewed By: spicyj\n\nDifferential Revision: D4767428\n\nfbshipit-source-id: 77c80c0135641ab46f9dce2763f27499a96373a0",
      "tree": {
        "sha": "884fa5a813f385add15fb240726e0eb7b1e7237a",
        "url": "https://api.github.com/repos/facebook/react-native/git/trees/884fa5a813f385add15fb240726e0eb7b1e7237a"
      },
      "url": "https://api.github.com/repos/facebook/react-native/git/commits/848593c0f0b75da9ff342596908c5091a2cd1430",
      "comment_count": 0
    },
    "url": "https://api.github.com/repos/facebook/react-native/commits/848593c0f0b75da9ff342596908c5091a2cd1430",
    "html_url": "https://github.com/facebook/react-native/commit/848593c0f0b75da9ff342596908c5091a2cd1430",
    "comments_url": "https://api.github.com/repos/facebook/react-native/commits/848593c0f0b75da9ff342596908c5091a2cd1430/comments",
    "author": {
      "login": "bvaughn",
      "id": 29597,
      "avatar_url": "https://avatars3.githubusercontent.com/u/29597?v=3",
    },
    "committer": {
      "login": "facebook-github-bot",
      "id": 6422482,
      "avatar_url": "https://avatars0.githubusercontent.com/u/6422482?v=3",

    },
    "parents": [
      {
        "sha": "30778b62eade6dc9add5880220af6242be2b7af8",
        "url": "https://api.github.com/repos/facebook/react-native/commits/30778b62eade6dc9add5880220af6242be2b7af8",
        "html_url": "https://github.com/facebook/react-native/commit/30778b62eade6dc9add5880220af6242be2b7af8"
      }
    ]
  }
it('renders the CommitItem from snapshot', () => {
  expect(renderer.create(
    <CommitItem {...commit}/>
  )).toMatchSnapshot()
})