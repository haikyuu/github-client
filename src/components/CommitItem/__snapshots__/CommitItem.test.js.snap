// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CommitItem from snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "paddingHorizontal": 4,
      "paddingVertical": 8,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://avatars3.githubusercontent.com/u/29597?v=3",
      }
    }
    style={
      Object {
        "borderRadius": 4,
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "borderBottomWidth": 0.2,
        "flex": 1,
        "marginLeft": 4,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      numberOfLines={1}
      style={
        Object {
          "fontSize": 12,
          "fontWeight": "bold",
          "paddingBottom": 4,
        }
      }
    >
      Moved takeSnapshot method from UIManager to ReactNative
      
      Reviewed By: spicyj
      
      Differential Revision: D4767428
      
      fbshipit-source-id: 77c80c0135641ab46f9dce2763f27499a96373a0
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 12,
        }
      }
    >
      Brian Vaughn committed with Facebook Github Bot 9 hours ago
    </Text>
  </View>
</View>
`;
